discovery-provider/plugins/pedalboard/apps/relay/src/routes/relay.ts

```diff
@@ -6,26 +6,28 @@ import {
 } from "../types/relay";
 import { SharedData } from "..";
 import { relayTransaction } from "../txRelay";
-import { IncomingHttpHeaders } from "http";
 import { FastifyReply } from "fastify";
+import {
+  handleError,
+  isError,
+} from "../error";
+import { logger } from "../logger";
 
 export const relayHandler = async (
   app: App<SharedData>,
   headers: RelayRequestHeaders,
   req: RelayRequestType,
   rep: FastifyReply
-): Promise<RelayResponseType> => {
-  try {
-    const { receipt } = await relayTransaction(app, headers, req, rep);
+): Promise<RelayResponseType | undefined> => {
+  const relay = await relayTransaction(app, headers, req, rep);
+  if (!isError(relay)) {
+    const { receipt } = relay;
```

Comment:
confirming js won't get confused w/ the naming of receipt twice here and line 26-29?